events/Alliance of Dwarves and Men.txt

```diff
@@ -16886,7 +16886,7 @@ country_event = {
 		set_variable = { which = alliance_dwarves_men_economic_union_location value = 0 }
 		set_variable = { which = alliance_dwarves_men_political_union_location value = 0 }
 		
-		# Call to Arms
+		# Call to Arms {
```

Comment:
The issue is actually the `# Calculate the number of options that should be shown` on 16878 is missing the open bracket